shell/platform/windows/flutter_windows_internal.h

```diff
@@ -14,6 +14,32 @@ extern "C" {
 // Declare functions that are currently in-progress and shall be exposed to the
 // public facing API upon completion.
 
+// Properties for configuring a Flutter view controller.
+typedef struct {
+  // The view's initial width.
+  int width;
+
+  // The view's initial height.
+  int height;
+
+} FlutterDesktopViewControllerProperties;
+
+// Creates a view for the given engine.
+//
+// The |engine| will be started if it is not already running.
+//
+// The caller owns the returned reference, and is responsible for calling
+// |FlutterDesktopViewControllerDestroy|. Returns a null pointer in the event of
+// an error.
+//
+// Unlike |FlutterDesktopViewControllerCreate|, this does *not* take ownership
```

Comment:
We might want to add a similar paragraph to `FlutterDesktopViewControllerCreate` as well, so that they can refer to each other. And also we might want to mention that:

```
// In an app with multiple view controllers, it is highly recommend to use
// `FlutterDesktopEngineCreateViewController` for all view controllers and avoid
// `FlutterDesktopViewControllerCreate`.
```

Reply:
We'll definitely want a blurb like this eventually, however, `FlutterDesktopEngineCreateViewController` isn't fully public yet. We'll add this comment when we make `FlutterDesktopEngineCreateViewController` public.